### ISSUE TITLE
Colored water does not water log stairs

### ISSUE THREAD

**Issue by weegeeluigi** [Exploding-Creeper/Hyper-Lighting#19]
**Describe the bug**
Colored water, glowing or non glowing, does not water log stairs like regular water.

**To Reproduce**
1. Place any stair.
2. Water log it with normal water.  It works.
3. Place any stair again.
4. This time use colored water (IE, Cyan Glowing Colored Water) and try to water log stair
5. Stair won't water log and water will be placed outside of the stair

**Expected behavior**
It works just like normal H2O

**Screenshots**
![2022-06-30_17 38 35](https://user-images.githubusercontent.com/7762155/176718917-c79d8541-c808-4c9a-ad9c-d48a2469a804.png)

1.18.2
Fabric Loader 0.13.3
Fabric Api 56.1
Mod Version 1.2.3

**Additional context**

No crash report applicable

**Comment by hypherionmc** [comment]
As noted in the colored water documentation page, colored water does not support waterlogging

Minecraft doesn't really allow much options for custom fluids to fo things like that. You basically have to use a ton of hacks.

For Hyper Lighting 2, which we're currently working on, I'll try to add it in somehow, without having to rely on hacks

**Comment by weegeeluigi** [comment]
I'm blind I didn't see there was an actual docs site outlining that.  So Minecraft being minecraft, shame because the glowing colored water would be so good for mystic/ritual builds.

**Comment by hypherionmc** [comment]
Yeah, the colored water still lacks a bit from vanilla water, but it's not a "no I won't do this". I just have to find a way that doesn't cause incompatibilities with 200 other mods :)
